### PR TITLE
[TA-LiveTest] use RECORD mode for recording

### PR DESCRIPTION
**Pull request by mssfang**
# Description

(1) A fixed for TA live tests failed for 
A recording policy can only be requested in RECORD mode.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2694604&view=ms.vss-test-web.build-test-results-tab&runId=40357490&paneView=debug&resultId=100367

(2) Dynamically Azure TA resource allocation failed for 

Status code 401, "{"error":{"code":"401","message":"Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource."

Solution: to add a longer wait time.

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java [hunk 1]
```diff
@@ -127,7 +128,8 @@ public class TextAnalyticsAsyncClientTest extends TextAnalyticsClientTestBase {
     private TextAnalyticsAsyncClient client;
 
     @BeforeAll
-    static void beforeAll() {
+    static void beforeAll() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(180);
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java:132]: Was this required for all or just one of the tests?

**Review by samvaity — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java:132]: All tests should start after the resource is dynamically allocated.

**Pull request merged**